Proof SERVICER.agc #636

KouroshSimpkins opened the pull request:
Checked pages 857 to 864 Inclusive, and will be continuing on SERVICER.agc.

wopian commented:
Hey @KouroshSimpkins, I didn't see the questions asked in the commit description at the time.

>although unsure about how to handle the newlines in the comment block on page 869

Consecutive line breaks without a line number should be a maximum of 2 line breaks. https://github.com/chrislgarry/Apollo-11/blob/master/CONTRIBUTING.md#line-breaks

In the case of R0383 and the surrounding line breaks, that would be:

- `[tab]EXIT IS VIA Q.`
- 2 empty lines above R0383
- R0383 itself as an empty line
- 1 empty line below R0383
- `INPUT`

>as well as when there is a "+x" in the righmost column, how should they be indented?

Those should be `[tab]+x[tab]INSTRUCTION`. Ensure you're using tab formatting in your editor and have tab width set to 8 https://github.com/chrislgarry/Apollo-11/blob/master/CONTRIBUTING.md#formatting

![image](https://user-images.githubusercontent.com/3440094/95004446-05ec0100-05e3-11eb-8f6e-3fbb72808d66.png)

KouroshSimpkins commented:
Hi there @wopian, that's my bad I should've probably asked them as a comment in this PR, obviously I forgot because it was an update after I'd opened the PR.

Thanks for clearing up those questions I had though, I did think the empty lines would stack, but wasn't sure. I've implemented the changes to reflect your responses. I'll continue to update this PR as I move through SERVICER.agc, and I'll copy my commit descriptions into comments if it makes things easier?

> @KouroshSimpkins
Update SERVICER.agc … d6168d9
Updated Parts of SERVICER to reflect the assistance provided with reference to large comment block on page 869 and +n in margins. Will continue to move through this (hopefully at about a page per day, maybe faster.) But have exams at the moment so will be a little slow. -KS

More updates soon
-KS

wopian commented:
> I'll continue to update this PR as I move through SERVICER.agc, and I'll copy my commit descriptions into comments if it makes things easier?

If you need a question answered it'd be best to copy it as a comment that mentions me (I'll see it much quicker among my other notifications) otherwise the commit descriptions are fine 👍 

Feel free to go at your own pace - it's a lot of pages to go through

chrislgarry commented:
@KouroshSimpkins is this still in draft?

KouroshSimpkins commented:
@chrislgarry unfortunately yes, I'm about 65-70% done though.

KouroshSimpkins commented:
Okay I'm now happy with what I've done in this proofing, fixed everything that I could find. Hopefully it all looks good!

wopian commented:
Thank you @KouroshSimpkins.

I took December off from Open Source development and will be back reviewing the other PRs I was unable to get through in October/November and yours this month onwards 👍
